### ISSUE TITLE
Large VMSS with singlePlacementGroup=false behind Public Load Balancer SKU=Standard has no outbound connection

### ISSUE THREAD

**Issue by evertonmc** [Azure/vm-scale-sets#45]
When deploying a (planned to be) large VMSS inside a vnet, this [doc ](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-placement-groups#converting-an-existing-scale-set-to-span-multiple-placement-groups) recommends to set singlePlacementGroup=false in order to support >100 VMs.
This kind of VMSS (100+) can only get load balanced from a LB-Standard ([preview](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-standard-overview#enterprisescale)).
Nevertheless, when using both settings, the instances are not able to get outbound traffic no matter what NSG rules a try.

Deploying the same setup with _Basic LB_ **or** _singlePlacementGroup_ recovers the outbound traffic.

**Comment by gbowerman** [comment]
@christiankuhtz @gatneil can you take a look at this?

**Comment by gatneil** [comment]
@evertonmc did you add PIP per VM or load balancing rules from the public LB? With Standard LB, you must have at least one of these to get outbound connectivity. Sample template here: https://github.com/Azure/vm-scale-sets/blob/master/preview/zones/multizone.json. This behavior is documented here: https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-standard-overview#outboundconnections

Let me know if you have further questions! :)

**Comment by evertonmc** [comment]
Hi. I have a PIP (Standard) bound to the LB (also Standard). You can reproduce this issue by deploying your template with singlePlacementGroup = false

**Comment by gatneil** [comment]
Can you share your template? A PIP on the LB is not enough. You must either have a PIP per VM or load balancing rules from the public LB to the scale set. This is new behavior in Standard Load Balancer

**Comment by evertonmc** [comment]
yes I can, on monday. I am currently using the Standard LB successfully. The problem is, that I can't deactivate singlePlacementGroup in order to scale to > 100 VMs

**Comment by evertonmc** [comment]
https://gist.github.com/evertonmc/5550de54b03940745aa683623155c289

**Comment by evertonmc** [comment]
Anything new here?
